CHANGELOG.rst

```diff
@@ -5,6 +5,24 @@ Change Log
    This file loosely adheres to the structure of https://keepachangelog.com/,
    but in reStructuredText instead of Markdown.
 
+2023-05-10
+**********
+
+Fixed
+=====
+
+- Add missing ``docs`` Makefile target to ``cookiecutter-django-ida`` and suppressed long-line lint as appropriate
+
+Changed
+=======
+
+- Use short version of ``BROWSER`` script in django-ida Makefile to match others
```

Comment:
Might also add "removed unused files in django-ida, django-app, and xblock"

Reply:
Added something like this